packages/webapp/src/components/Noun/Noun.module.css

```diff
@@ -82,9 +93,9 @@
 }
 
 @media only screen and (min-width: 1025px) {
-    .img {
-        width:  500px;
-        max-width: 500px;
+    /* Single & Double Noun */
+    .nounWrapper .img, .nounsWrapper .img, {
```

Comment:
The trailing commas here and on line #82  are actually breaking the width for me. This may be a browser compatibility thing (although saw it in a couple browsers I tested).